src/uikit.cs

```diff
@@ -18235,6 +18337,48 @@ interface UIDocumentBrowserAction {
 		[Export ("supportsMultipleItems")]
 		bool SupportsMultipleItems { get; set; }
 	}
+
+	interface IUIFocusItemContainer {}
+	[iOS (12,0), TV (12,0), NoWatch]
+	[Protocol]
+	interface UIFocusItemContainer
+	{
+		[Abstract]
+		[Export ("coordinateSpace")]
+		IUICoordinateSpace CoordinateSpace { get; }
+
+		[Abstract]
+		[Export ("focusItemsInRect:")]
+		IUIFocusItem[] GetFocusItems (CGRect rect);
+	}
+
+	[iOS (12,0), TV(12,0), NoWatch]
+	[Protocol]
+	interface UIFocusItemScrollableContainer : UIFocusItemContainer
+	{
+		[Abstract]
+		[Export ("contentOffset", ArgumentSemantic.Assign)]
+		CGPoint ContentOffset { get; set; }
+
+		[Abstract]
+		[Export ("contentSize")]
+		CGSize ContentSize { get; }
+
+		[Abstract]
+		[Export ("visibleSize")]
+		CGSize VisibleSize { get; }
+	}
+
+	[iOS (8,0), TV (8,0), NoWatch] // it was added on 8,0, but was not binded and the method was added in 12,0
```

Comment:
tvOS started with 9.0 so remove it's attribute (which means it's always been available)